packages/salesforcedx-vscode-core/src/settings/sfdxCoreSettings.ts

```diff
@@ -54,6 +55,10 @@ export class SfdxCoreSettings {
     return this.getConfigValue<boolean>(PUSH_OR_DEPLOY_ON_SAVE_ENABLED, false);
   }
 
+  public getRetrieveTestCodeCoverage(): boolean {
```

Comment:
This is good, lets also update `forceApexTestRun.ts` and `forceApexTestRunCodeAction.ts` to use it instead of having `sfdxCoreSettings.getConfiguration().get('retrieve-test-code-coverage')` there.